### PR TITLE
Docs: add RBAC for library panels

### PR DESCRIPTION
**Pull request by imatwawana**
This PR updates the library panels page to include RBAC capability. Updates RBAC fixed role, custom role, actions, scopes with library panel information.

### CODE REVIEW

File: docs/sources/administration/roles-and-permissions/access-control/custom-role-actions-scopes/index.md [hunk 1]
```diff
@@ -88,6 +88,10 @@ The following list contains role-based access control actions.
...
+| `library.panels:read`                | `folders:*` <br> `folders:uid:*` <br> `library.panels:*` <br> `library.panels:uid:*`    | Read one or more library panels token.                                                                                                                                                                              |
+| `library.panels:write`               | `folders:*` <br> `folders:uid:*` <br> `library.panels:*` <br> `library.panels:uid:*`    | Update one or more library panels                                                                                                                                                                                   |
+| `library.panels:delete`              | `folders:*` <br> `folders:uid:*` <br> `library.panels:*` <br> `library.panels:uid:*`    | Delete one or more library panelstoken.                                                                                                                                                                             |
```
Comment (imatwawana) [inline docs/sources/administration/roles-and-permissions/access-control/custom-role-actions-scopes/index.md:94]:
```suggestion
| `library.panels:delete`              | `folders:*` <br> `folders:uid:*` <br> `library.panels:*` <br> `library.panels:uid:*`    | Delete one or more library panels.                                                                                                                                                                             |
```

File: docs/sources/administration/roles-and-permissions/access-control/custom-role-actions-scopes/index.md [hunk 2]
```diff
@@ -88,6 +88,10 @@ The following list contains role-based access control actions.
...
 | `ldap.user:sync`                     | n/a                                                                                     | Sync users via LDAP.                                                                                                                                                                                                |
+| `library.panels:create`              | `folders:*` <br> `folders:uid:*`                                                        | Create a library panel in one or more folders and their subfolders.                                                                                                                                                 |
+| `library.panels:read`                | `folders:*` <br> `folders:uid:*` <br> `library.panels:*` <br> `library.panels:uid:*`    | Read one or more library panels token.                                                                                                                                                                              |
```
Comment (imatwawana) [inline docs/sources/administration/roles-and-permissions/access-control/custom-role-actions-scopes/index.md:92]:
```suggestion
| `library.panels:read`                | `folders:*` <br> `folders:uid:*` <br> `library.panels:*` <br> `library.panels:uid:*`    | Read one or more library panels.                                                                                                                                                                              |
```

File: docs/sources/administration/roles-and-permissions/access-control/custom-role-actions-scopes/index.md [hunk 3]
```diff
@@ -88,6 +88,10 @@ The following list contains role-based access control actions.
...
+| `library.panels:write`               | `folders:*` <br> `folders:uid:*` <br> `library.panels:*` <br> `library.panels:uid:*`    | Update one or more library panels                                                                                                                                                                                   |
```
Comment (imatwawana) [inline docs/sources/administration/roles-and-permissions/access-control/custom-role-actions-scopes/index.md:93]:
```suggestion
| `library.panels:write`               | `folders:*` <br> `folders:uid:*` <br> `library.panels:*` <br> `library.panels:uid:*`    | Update one or more library panels.                                                                                                                                                                                   |
```

File: docs/sources/administration/roles-and-permissions/access-control/rbac-fixed-basic-role-definitions/index.md [hunk 4]
```diff
@@ -23,13 +23,13 @@ The following tables list permissions associated with basic and fixed roles.
...
... Admin         | `fixed:reports:reader`<br>`fixed:reports:writer`<br>`fixed:datasources:reader`<br>`fixed:datasources:writer`<br>`fixed:organization:writer`<br>`fixed:datasources.permissions:reader`<br>`fixed:datasources.permissions:writer`<br>`fixed:teams:writer`<br>`fixed:dashboards:reader`<br>`fixed:dashboards:writer`<br>`fixed:dashboards.permissions:reader`<br>`fixed:dashboards.permissions:writer`<br>`fixed:dashboards.public:writer`<br>`fixed:folders:reader`<br>`fixed:folders:writer`<br>`fixed:folders.permissions:reader`<br>`fixed:folders.permissions:writer`<br>`fixed:alerting:writer`<br>`fixed:apikeys:reader`<br>`fixed:apikeys:writer`<br>`fixed:alerting.provisioning.secrets:reader`<br>`fixed:alerting.provisioning:writer`<br>`fixed:datasources.caching:reader`<br>`fixed:datasources.caching:writer`<br>`fixed:dashboards.insights:reader`<br>`fixed:datasources.insights:reader`<br>`fixed:plugins:writer`<br>`fixed:library.panels:creator`<br>`fixed:library.panels:reader`<br>`fixed:library.panels:general.reader`<br>`fixed:library.panels:writer`<br>`fixed:library.panels:general. | Default [Grafana organization administrator]({{< relref "../#basic-roles" >}}) assignments.                |
```
Comment (imatwawana) [inline docs/sources/administration/roles-and-permissions/access-control/rbac-fixed-basic-role-definitions/index.md:29]:
```suggestion
| Admin         | `fixed:reports:reader`<br>`fixed:reports:writer`<br>`fixed:datasources:reader`<br>`fixed:datasources:writer`<br>`fixed:organization:writer`<br>`fixed:datasources.permissions:reader`<br>`fixed:datasources.permissions:writer`<br>`fixed:teams:writer`<br>`fixed:dashboards:reader`<br>`fixed:dashboards:writer`<br>`fixed:dashboards.permissions:reader`<br>`fixed:dashboards.permissions:writer`<br>`fixed:dashboards.public:writer`<br>`fixed:folders:reader`<br>`fixed:folders:writer`<br>`fixed:folders.permissions:reader`<br>`fixed:folders.permissions:writer`<br>`fixed:alerting:writer`<br>`fixed:apikeys:reader`<br>`fixed:apikeys:writer`<br>`fixed:alerting.provisioning.secrets:reader`<br>`fixed:alerting.provisioning:writer`<br>`fixed:datasources.caching:reader`<br>`fixed:datasources.caching:writer`<br>`fixed:dashboards.insights:reader`<br>`fixed:datasources.insights:reader`<br>`fixed:plugins:writer`<br>`fixed:library.panels:creator`<br>`fixed:library.panels:reader`<br>`fixed:library.panels:general.reader`<br>`fixed:library.panels:writer`<br>`fixed:library.panels:general.writer` | Default [Grafana organization administrator]({{< relref "../#basic-roles" >}}) assignments.                |
```

**Review by kaydelaney — APPROVED** [review]

File: docs/sources/dashboards/build-dashboards/manage-library-panels/index.md [hunk 5]
```diff
@@ -91,3 +95,8 @@ Delete a library panel when you no longer need it.
 1. Click **Dashboards** in the left-side menu.
 1. Click **Library panels**.
 1. Click the delete icon next to the library panel name.
+
+{{% docs/reference %}}
+[rbac]: "/docs/grafana/ -> /docs/grafana/<GRAFANA VERSION>/dashboards/build-dashboards/manage-library-panels"
+[rbac]: "/docs/grafana-cloud/ -> /docs/grafana/<GRAFANA VERSION>/dashboards/build-dashboards/manage-library-panels"
```
Comment (jdbaldry) [inline docs/sources/dashboards/build-dashboards/manage-library-panels/index.md:101]:
```suggestion
[rbac]: "/docs/grafana/ -> /docs/grafana/<GRAFANA_VERSION>/dashboards/build-dashboards/manage-library-panels"
[rbac]: "/docs/grafana-cloud/ -> /docs/grafana-cloud/visualizations/dashboards/build-dashboards/manage-library-panels"
```

Although I've realized these are linking to the current page. Is that intended?

File: docs/sources/dashboards/build-dashboards/manage-library-panels/index.md [hunk 6]
```diff
@@ -91,3 +95,8 @@ Delete a library panel when you no longer need it.
 1. Click **Dashboards** in the left-side menu.
 1. Click **Library panels**.
 1. Click the delete icon next to the library panel name.
+
+{{% docs/reference %}}
+[rbac]: "/docs/grafana/ -> /docs/grafana/<GRAFANA VERSION>administration/roles-and-permissions/access-control"
+[rbac]: "/docs/grafana-cloud/ -> /docs/grafana/<GRAFANA VERSION>administration/roles-and-permissions/access-control"
```
Comment (imatwawana) [inline docs/sources/dashboards/build-dashboards/manage-library-panels/index.md:101]:
```suggestion
[rbac]: "/docs/grafana/ -> /docs/grafana/<GRAFANA VERSION>/administration/roles-and-permissions/access-control"
[rbac]: "/docs/grafana-cloud/ -> /docs/grafana/<GRAFANA VERSION>/administration/roles-and-permissions/access-control"
```

**Review by jdbaldry — APPROVED** [review]

**Pull request merged**